tests/lib/rules-preprocessor/gjs-gts-processor-test.js

```diff
@@ -254,3 +255,30 @@ describe('line/col numbers should be correct', () => {
     });
   });
 });
+
+describe('template tag failing test repro', () => {
+  it('should break', async () => {
```

Comment:
Can you update these to be more descriptive now that the test is passing? 
Also the template below "this breaks if there isn't a newline above template tag" 😄

Reply:
great call